### ISSUE TITLE
Isoseq3 refine for LIMA discarded reads

### ISSUE THREAD

**Issue by juheon** [PacificBiosciences/pbbioconda#150]
Hi, 
Using Iso-seq3, I wanted to look into how many of reads that are discarded during LIMA are full-length transcripts. I obtained discarded reads by LIMA using "--dump-removed". (lima_discarded.bam). 

First, to get rid of concatemer in lima.discarded.bam, I tried 
`isoseq3 refine lima_discarded.bam primer lima_discarded.refine.bam`
and it said "bc, bq, bx" tags are required.
To go around this issue, I used "lima.report" to add bam tags. I used "ScoreCombined", "IdxsCombined",  and "ClipsCombined" to fill in bq, bc, and bx tags, respectively. (lima_discarded.withBAMTags.bam)

Then I ran
`isoseq3 refine lima_discarded.withBAMTags.bam primer lima_discarded.withBAMTags.refine.bam`

But it failed and the error message was
`>|> 20190509 21:56:50.058 -|- FATAL      -|- operator() -|- 0x7f0ca0310740|| -|- caught exception: "bad numeric conversion: positive overflow"
>|> 20190509 21:56:50.058 -|- FATAL      -|- operator() -|- 0x7f0ca0310740|| -|- caught SIGABRT
Aborted (core dumped)`

Could you give me more information so that I go around this error?

**Comment by armintoepfer** [comment]
You cannot hack the system by using files that were not intended for post processing. What is discarded can't be post processed.

**Comment by juheon** [comment]
I am not hacking your code but trying to use your code. 

I prepared PacBio cDNA libraries of human cancer cell lines and found that for LIMA discarded reads, I found some of those actually mapped to GENCODE transcript annotation. I hypothesize this is due to sequence similarity of PCR forward/reverse primers that I used. If I can salvage more reads by changing PCR forward/reverse primers I will modify my protocol.

For this purpose, I tried to use isoseq3 refine for LIMA-discarded reads because LIMA-discard reads contain concatemers. 

I want to align non-concatemer LIMA-discarded reads.
